iina/uk.lproj/Localizable.strings

```diff
@@ -96,10 +96,10 @@
 "menu.audio_delay_down" = "Затримка звуку на –%.1f с";
 "menu.audio_delay_up" = "Затримка звуку на +%.1f с";
 "menu.exit_fullscreen" = "Вийти з повноекранного режиму";
-"menu.exit_pip" = "Вийти з режиму «Кадр у кадрі»";
+"menu.exit_pip" = "Вийти з режиму «Картинка в картинці»";
 "menu.fullscreen" = "Увійти до повноекранного режиму";
 "menu.pause" = "Пауза";
-"menu.pip" = "Режим «Кадр у кадрі»";
+"menu.pip" = "Режим «Картинка в картинці»";
```

Comment:
Wrong. The red one is used in official Ukrainian localisation of macOS apps (eg., Safari web browser).
<img width="292" alt="Screenshot 2019-05-06 16 38 04" src="https://user-images.githubusercontent.com/36358174/57229233-3ca09f00-701e-11e9-9104-12819044c9e9.png">